.update-copyright.conf

```diff
@@ -0,0 +1,17 @@
+[project]
+name: lesson-template
```

Comment:
Can `update-copyright.py` infer the name of the repo by looking at its 
Git origin remote instead?